docs/learn/tutorials/versioned/samza-sql.md

```diff
@@ -0,0 +1,129 @@
+---
+layout: page
+title: How to use Samza SQL
+---
+<!--
+   Licensed to the Apache Software Foundation (ASF) under one or more
+   contributor license agreements.  See the NOTICE file distributed with
+   this work for additional information regarding copyright ownership.
+   The ASF licenses this file to You under the Apache License, Version 2.0
+   (the "License"); you may not use this file except in compliance with
+   the License.  You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+-->
+
+There are couple of ways to use Samza SQL
+
+1. Run Samza SQL on your local machine.
+2. Run Samza SQL on YARN.
+
+# Running Samza SQL on your local machine  
+
+Samza SQL console tool documented [here](https://github.com/srinipunuru/samza-sql-tools/blob/master/README.md) uses Samza standalone to run the Samza SQL on your local machine. This is the quickest way to play with Samza SQL. Please follow the instructions [here](https://github.com/srinipunuru/samza-sql-tools/blob/master/README.md) on how to use the Samza SQL console to run stream processing jobs on your local machine using SQL.
```

Comment:
Why don't we have samza-sql-tools in Samza repo?

Reply:
@nickpan47 Please see the reasoning above in description. Depends on what is the deadline for 0.14 release. Merging the samza-sql-tools into Samza might take atleast a week or more based on my experience with Samza PR round trip time.